src/problem/constr/constr_NBOUND.c

```diff
@@ -598,8 +598,8 @@ void CONSTR_NBOUND_eval_step(Constr* c, Branch* br, int t, Vec* values) {
 	  BUS_has_flags(bus,FLAG_VARS,BUS_VAR_VMAG)) {
 
 	u = VEC_get(values,BUS_get_index_v_mag(bus,t));
-	umax = BUS_get_v_max(bus);
-	umin = BUS_get_v_min(bus);
+	umax = BUS_get_v_max_reg(bus);
+	umin = BUS_get_v_min_reg(bus);
```

Comment:
- The constraints
"voltage regulation by transformer"
"voltage regulation by shunt"
enforce band regulation and we should include in the docs that that v_max_reg and v_min_reg are used. 

- The constraint "voltage regulation by gens" enforces setpoint regulation and hence does not involve limits.

- The function "voltage magnitude regularization" does not involve limits and is not related to regulation.